antarest/login/web.py

```diff
@@ -373,6 +330,34 @@ def get_all_bots() -> Any:
         )
         return jsonify([b.to_dict() for b in bots]), 200
 
+    @bp.route("/bots/infos/<int:id>", methods=["GET"])
+    @auth.protected()
+    def bots_get_infos(id: int) -> Any:
+        params = RequestParameters(user=Auth.get_current_user())
+        b = service.get_bots_infos(id, params)
+        if b:
+            return jsonify(b.to_dict())
+        else:
+            return "", 404
```

Comment:
remove this and aggregate the basic identity information with specific bot properties in the method get_bot